test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1770,7 +1766,6 @@ private void CompareResults(string leftColumnName, string rightColumnName, IData
             var rightColumn = right.Schema[rightColumnName];
             var leftType = leftColumn.Type.GetItemType();
             var rightType = rightColumn.Type.GetItemType();
-            Assert.Equal(leftType, rightType);
```

Comment:
Review: Types don't necessarily have to be equal. For example, an ML.NET type can be Key{uint32} and uint32 in onnx.